os_migrate/plugins/modules/import_workload_export_volumes.py

```diff
@@ -517,10 +522,11 @@ def _export_volumes_from_converter(self):
 
 def run_module():
     argument_spec = openstack_full_argument_spec(
+        auth=dict(type='dict', no_log=True, required=True),
         data=dict(type='dict', required=True),
         boot_volume_prefix=dict(type='str', default=None),
         conversion_host=dict(type='dict', required=True),
-        ssh_key_path=dict(type='str', required=True),
+        ssh_key_path=dict(type='str', required=True, no_log=True),
```

Comment:
Ditto

Reply:
Now reverted in https://github.com/os-migrate/os-migrate/pull/484/commits/dcb6ba052d85e887f34570d66b489a6b7a0a09da